app/reducers/loginReducer.js

```diff
@@ -0,0 +1,28 @@
+import { LOGIN_REQUEST, LOGIN_SUCCESS, LOGIN_ERROR } from '../actions/actionTypes';
+import initialState from './initialState';
+
+export const login = (state = initialState.login, action) => {
```

Comment:
I think you are repeating too much `login` prefix. It should be better something like this:
In case of ACTIONS:
actionTypes

```
export const LOGIN_USER_SUCCESS = 'LOGIN_USER_SUCCESS';
export const LOGIN_USER_ERROR = 'LOGIN_USER_ERROR';
```
loginActions.js => userSessionActions.js

loginReducer.js => userSessionReducer.js

```
export userSession = (state = initialState.userSession, action) => {
```

Reply:
You might be right, honestly I am not sure. I used the same prefix everywhere so any developer can quickly realize all that code belongs to the same functionality.

Reply:
👍